hack/BUILD

```diff
@@ -39,11 +39,19 @@ sh_test(
 #    tags = ["manual"],
 #)
 
+sh_test(
```

Comment:
We can even put this file in the actual `pkg/util` directory itself. There's no real need for it to live here. Then it can just have `data = glob(["*.go"])`.